src/Umbraco.Core/Services/IRelationService.cs

```diff
@@ -215,26 +215,70 @@ public interface IRelationService : IService
         /// <param name="pageIndex"></param>
         /// <param name="pageSize"></param>
         /// <param name="totalChildren"></param>
-        /// <returns></returns>
+        /// <returns>An enumerable list of <see cref="IUmbracoEntity"/></returns>
         IEnumerable<IUmbracoEntity> GetPagedParentEntitiesByChildId(int id, long pageIndex, int pageSize, out long totalChildren, params UmbracoObjectTypes[] entityTypes);
 
+        /// <summary>
+        /// Returns paged parent entities for a related child id
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="pageIndex"></param>
+        /// <param name="pageSize"></param>
+        /// <param name="totalChildren"></param>
+        /// <param name="relationTypes">A list of relation types to filter</param>
+        /// <returns>An enumerable list of <see cref="IUmbracoEntity"/></returns>
+        IEnumerable<IUmbracoEntity> GetPagedParentEntitiesByChildId(int id, long pageIndex, int pageSize, out long totalChildren, string[] relationTypes, params UmbracoObjectTypes[] entityTypes);
```

Comment:
Same here.. We need to add these methods on a new interface.

Reply:
On it 👍